src/contentscript/overlay/root/components/AlertConfirmPopup/index.tsx

```diff
@@ -0,0 +1,67 @@
+import cn from 'classnames'
+import React, { useEffect } from 'react'
+import NO_LOGO from '../../../../../common/resources/no-logo.png'
+import { LinkifyText } from '../LinkifyText'
+import stylesNotifications from '../OverlayToolbar/OverlayToolbar.module.scss'
+import stylesAlerts from './AlertConfirmPopup.module.scss'
+import { ModalProps } from './contexts/ModalContext/ModalContext'
+
+const AlertConfirmPopup = (props: { payload: ModalProps }) => {
+  const { payload } = props
+  const { title, message, icon, type, onResolve } = payload
+
+  useEffect(() => {
+    const handleKeydown = (e) => {
+      if (e.key === 'Enter') {
+        onResolve(true)
+      }
+    }
+    window.addEventListener('keydown', handleKeydown)
+    return () => window.removeEventListener('keydown', handleKeydown)
+  }, [onResolve])
```

Comment:
It seems that if there are multiple modal windows, then they will be resolved together by one pressing Enter. Is it expected behaviour?